machines/4x4_dmc/Makefile.machine.include

```diff
@@ -3,6 +3,7 @@ BSG_MACHINE_GLOBAL_Y                  = 5
 BSG_MACHINE_VCACHE_SET                = 128
 BSG_MACHINE_VCACHE_WAY                = 8
 BSG_MACHINE_VCACHE_BLOCK_SIZE_WORDS   = 8
+BSG_MACHINE_VCACHE_DMA_DATA_WIDTH     = 32
```

Comment:
Is there a reason this wasn't added to 9x15? Are we not modifying 9x15?

Reply:
9x15 will be dropped.